src/tooling/piral-cli/src/apps/new-pilet.test.ts

```diff
@@ -15,12 +15,29 @@ describe('New Pilet Command', () => {
     await newPilet(dir, {
       install: false,
       source: 'piral@latest',
+      registry: 'https://someFakeRegistry.com',
     });
     expect(existsSync(resolve(dir, 'node_modules/piral/package.json'))).toBeTruthy();
     expect(existsSync(resolve(dir, 'package.json'))).toBeTruthy();
     expect(existsSync(resolve(dir, 'tsconfig.json'))).toBeTruthy();
     expect(existsSync(resolve(dir, 'src/index.tsx'))).toBeTruthy();
-    expect(existsSync(resolve(dir, '.npmrc'))).toBeFalsy();
+    expect(existsSync(resolve(dir, '.npmrc'))).toBeTruthy();
```

Comment:
Could we add this via another test?

We want at least one test to see that an npmrc was not added (default taken) and that one was added (custom taken).

Reply:
added own unit test for npmrc file creation